migrations/20240710084241-current-collective-views-update.js

```diff
@@ -0,0 +1,132 @@
+'use strict';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface) {
+    await queryInterface.sequelize.query(`
+      CREATE OR REPLACE VIEW "CurrentCollectiveTransactionStats" as (
+        SELECT
+          cts."id" as "CollectiveId",
+
+          COALESCE(cts."totalAmountReceivedInHostCurrency", 0) + COALESCE(t."totalAmountReceivedInHostCurrency", 0)
+            AS "totalAmountReceivedInHostCurrency",
+          COALESCE(cts."totalNetAmountReceivedInHostCurrency", 0) + COALESCE(t."totalNetAmountReceivedInHostCurrency", 0)
+            AS "totalNetAmountReceivedInHostCurrency",
+          COALESCE(cts."totalAmountSpentInHostCurrency", 0) + COALESCE(t."totalAmountSpentInHostCurrency", 0)
+            AS "totalAmountSpentInHostCurrency",
+          COALESCE(cts."totalNetAmountSpentInHostCurrency", 0) + COALESCE(t."totalNetAmountSpentInHostCurrency", 0)
+            AS "totalNetAmountSpentInHostCurrency",
+
+          cts."hostCurrency"
+
+        FROM "CollectiveTransactionStats" cts
+
+        LEFT JOIN LATERAL (
+          SELECT
+
+          SUM(t."amountInHostCurrency")
+            FILTER (WHERE t.type = 'CREDIT' AND t."kind" NOT IN ('PAYMENT_PROCESSOR_COVER'))
+            AS "totalAmountReceivedInHostCurrency",
+
+          SUM(
+            COALESCE(t."amountInHostCurrency", 0) +
+            COALESCE(t."platformFeeInHostCurrency", 0) +
+            COALESCE(t."hostFeeInHostCurrency", 0) +
+            COALESCE(t."paymentProcessorFeeInHostCurrency", 0) +
+            COALESCE(t."taxAmount" * t."hostCurrencyFxRate", 0)
+          )
+            FILTER (WHERE t.type = 'CREDIT' OR (t.type = 'DEBIT' AND t.kind IN ('HOST_FEE', 'PAYMENT_PROCESSOR_FEE') AND t."OrderId" IS NOT NULL))
```

Comment:
This is the only line that differs from https://github.com/opencollective/opencollective-api/blob/697ec9f9ce9927a7644c949130e49adb1ccb9596/migrations/20240202084241-current-collective-views-update.js#L76.

```diff
- FILTER (WHERE t.type = 'CREDIT' OR (t.type = 'DEBIT' AND t.kind IN ('HOST_FEE', 'PAYMENT_PROCESSOR_FEE')))
+ FILTER (WHERE t.type = 'CREDIT' OR (t.type = 'DEBIT' AND t.kind IN ('HOST_FEE', 'PAYMENT_PROCESSOR_FEE') AND t."OrderId" IS NOT NULL))
```